profile/profiler.go

```diff
@@ -427,6 +430,10 @@ func (c *CPUInfo) GetSibling(processor int) (int, error) {
 	}
 
 	core := c.sockets[proc.socket].cores[proc.core]
+	if len(core.processors) == 1 {
+		return -1, nil
```

Comment:
I guess `-1` means an error. The function should not return invalid values, instead, it should use the error return value and the caller needs to appropriately handle this error.

Reply:
Done